pulsar-metadata/src/test/java/org/apache/pulsar/metadata/LocalMemoryMetadataStoreTest.java

```diff
@@ -92,4 +75,15 @@ public void testSharedInstance() throws Exception {
             assertFalse(store1.exists("/test").join());
         });
     }
+
+    @Test
+    public void testPathValid() {
+        assertFalse(AbstractMetadataStore.isValidPath(null));
+        assertFalse(AbstractMetadataStore.isValidPath(""));
+        assertFalse(AbstractMetadataStore.isValidPath(" "));
+        assertFalse(AbstractMetadataStore.isValidPath("/"));
```

Comment:
Should be true?

Reply:
It's intentional false in previous code.
I think it's because all valid path comes with some prefix like "/admin/xxx", and the root node is not used.

Reply:
It seems that we have delete "/" operation in `testDeleteCluster(org.apache.pulsar.tests.integration.cli.ClusterMetadataTearDownTest)`.